lib/node_modules/@stdlib/lapack/base/dlacn2/test/test.dlacn2.js

```diff
@@ -0,0 +1,262 @@
+			break;
+		} else if ( KASE[ 0 ] === 1 ) {
+			/*
+			* -   If `KASE` == 1: `dlacn2` wants us to compute A * X and overwrite X.
+			* -   `dgemv` performs the matrix-vector product: work = A * X.
+			* -   A workspace array is necessary here because of the function signature of `dgemv`.
+			* -   `dcopy` copies work into X, so that the updated X is passed back to `dlacn2`.
+			*/
+			dgemv( 'row-major', 'no-transpose', 1, 1, 1.0, A, 1, X, 1, 0, work, 1 );
+			dcopy( 1, work, 1, X, 1 );
+		} else if ( KASE[ 0 ] === 2 ) {
+			/*
+			* -   If `KASE` == 2: `dlacn2` wants us to compute A ^ T * X and overwrite X.
+			* -   `dgemv` performs the matrix-vector product: work = A ^ T * X.
+			* -   A workspace array is necessary here because of the function signature of `dgemv`.
+			* -   `dcopy` copies work into X, so that the updated X is passed back to `dlacn2`.
+			*/
+			dgemv( 'row-major', 'transpose', 1, 1, 1.0, A, 1, X, 1, 0, work, 1 );
+			dcopy( 1, work, 1, X, 1 );
+		}
+	}
+
+	expectedEST = new Float64Array( [ 25.0 ] );
+	expectedV = new Float64Array( [ -25.0 ] );
+	expectedKASE = new Int32Array( [ 0 ] );
+	expectedISGN = new Int32Array( [ 0 ] );
+	expectedISAVE = new Int32Array( [ 1, 0, 0 ] );
+
+	t.deepEqual( KASE, expectedKASE, 'returns expected value' );
+	t.deepEqual( ISGN, expectedISGN, 'returns expected value' );
+	t.deepEqual( EST, expectedEST, 'returns expected value' );
+	t.deepEqual( V, expectedV, 'returns expected value' );
+	t.deepEqual( ISAVE, expectedISAVE, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns expected values when calculating the one-norm of a square matrix', function test( t ) {
+	var expectedISAVE;
+	var expectedKASE;
+	var expectedISGN;
+	var expectedEST;
+	var expectedV;
+	var ISAVE;
+	var KASE;
+	var ISGN;
+	var work;
+	var EST;
+	var X;
+	var V;
+	var A;
+
+	A = new Float64Array([
+		1.0, -2.0, 0.0, 0.0,
+		3.0, 4.0, -5.0, 0.0,
+		0.0, 6.0, 7.0, -8.0,
+		0.0, 0.0, 9.0, 10.0
+	]);
+
+	KASE = new Int32Array( 1 );
+	EST = new Float64Array( 1 );
+	ISGN = new Int32Array( 4 );
+	ISAVE = new Int32Array( 3 );
+	X = new Float64Array( 4 );
+	V = new Float64Array( 4 );
+
+	work = new Float64Array( 4 );
+
+	// Start the estimation with `KASE` = 0:
+	while ( true ) {
+		dlacn2( 4, V, X, ISGN, EST, KASE, ISAVE );
+
+		if ( KASE[ 0 ] === 0 ) {
+			// Estimation is complete:
+			break;
+		} else if ( KASE[ 0 ] === 1 ) {
+			/*
+			* -   If `KASE` == 1: `dlacn2` wants us to compute A * X and overwrite X.
+			* -   `dgemv` performs the matrix-vector product: work = A * X.
+			* -   A workspace array is necessary here because of the function signature of `dgemv`.
+			* -   `dcopy` copies work into X, so that the updated X is passed back to `dlacn2`.
+			*/
+			dgemv( 'row-major', 'no-transpose', 4, 4, 1.0, A, 4, X, 1, 0, work, 1 );
+			dcopy( 4, work, 1, X, 1 );
+		} else if ( KASE[ 0 ] === 2 ) {
+			/*
+			* -   If `KASE` == 2: `dlacn2` wants us to compute A ^ T * X and overwrite X.
```

Comment:
```suggestion
			* -   If `KASE` == 2: `dlacn2` wants us to compute A ^ T * X and overwrite X.
```
the `*` shouldn't be here